src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -22,13 +22,15 @@ public record CommandContext(
     String collection,
     boolean isVectorEnabled,
     CollectionSettings.SimilarityFunction similarityFunction,
-    EmbeddingService embeddingService) {
+    EmbeddingService embeddingService,
+    CollectionSettings.IndexingConfig indexingConfig) {
```

Comment:
I am planning to change this to pass `CollectionSettings`, instead of just pieces.